README.md

```diff
@@ -32,56 +32,26 @@ To attract more web developers we plan to build a JavaScript DSL for TEAL with T
+- Connection to an Algorand node. Follow our [infrastructure guide](https://github.com/scale-it/algo-builder/tree/master/infrastructure/README.md) for instructions how to setup a private network (using Algorand node binaries or docker based setup).
   NOTE: TEAL compilation requires Developer API to be enabled (`"EnableDeveloperAPI": true` in the node config.json).
-- Python 3.7+ (for PyTeal) with [pyteal](https://pypi.org/project/pyteal). Please read below how to install it.
-- Yarn `v3.1+` or NPM `v8.0+` (note: all our examples use yarn v3 workspace to link to the project algob build).
+- Python 3.7+ (for PyTeal) with [pyteal](https://pypi.org/project/pyteal). Please read below how to install it. We recommend to use `pipenv` with our `Pipfile` to stay always up to date.
+- Yarn `v3.1+` or NPM `v8.0+` or PNPM `v6.21+` (note: all our examples use yarn v3 workspaces).
 
-### Installation
-
-#### Pre requirements
-
-Make sure your `yarn global bin` directory is in your `$PATH\*\*.
-
-#### Installation from master
-
-We recommend installation from GitHub _master_ branch if you want to play with [templates](https://github.com/scale-it/algo-builder-templates) and [examples](https://github.com/scale-it/algo-builder/tree/master/examples) (all our example smart contracts are tested against _master_). The master branch corresponds to the latest released version.
-
-```
-git clone https://github.com/scale-it/algo-builder.git
-cd algo-builder
-yarn install
-yarn build
-cd packages/algob
-```
-
-##### Upgrading
-
-If you use installation from master, don't forget to **pull the latest changes** to not to miss the updates:
-
-```
-cd path/to/algo-builder
-git pull -p
-yarn install
-yarn build
-```
-
-#### Installation from released version
+### Algorand Node requirements
 
-To install `algob` globally in your system you can use:
+If you want to use private network then you need to install go-algorand (algod v3.2.0 or higher) or the Algorand Sandbox.
 
-- Using Yarn: `yarn global add @algo-builder/algob`
-- Using NPM: `npm install -g @algo-builder/algob`
+Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
 
-### Algorand Node requirements
+Follow our [infrastructure guide](https://github.com/scale-it/algo-builder/tree/master/infrastructure/README.md) for more details.
 
-- algod v2.1.6-stable or higher
+### Connecting to an external Algorand Node
 
-Make sure that the node you are connecting to has a `"EnableDeveloperAPI": true` option set in the `<node_data>/config.json`. This is required to compile smart contracts using REST / SDK.
+Instead of using a local node, you can connect to an external one by setting the network configuration in the [`algob.config.js`](guide/algob-config) file. More about the project setup and config file is described in the Quick Start section below.
```

Comment:
Should change to guide/algob-config to https://github.com/scale-it/algo-builder/blob/master/docs/guide/algob-config.md

Reply:
Oh, Thanks!